src/deploy/scope_managementgroup.ts

```diff
@@ -15,12 +15,12 @@ export async function DeployManagementGroupScope(azPath: string, validationOnly:
     }
     // create the parameter list
     const azDeployParameters = [
-        region ? `--location ${region}` : undefined,
+        region ? `--location "${region}"` : undefined,
         template ?
             template.startsWith("http") ? `--template-uri ${template}` : `--template-file ${template}`
             : undefined,
-        managementGroupId ? `--management-group-id ${managementGroupId}` : undefined,
-        deploymentName ? `--name ${deploymentName}` : undefined,
+        managementGroupId ? `--management-group-id "${managementGroupId}"` : undefined,
+        deploymentName ? `--name "${deploymentName}"` : undefined,
         parameters ? `--parameters ${parameters}` : undefined
```

Comment:
what if the value of a parameter needs to have spaces. Will user have to use double quotes and will that work?
For example `parameters: foo="bar baz"`?

Reply:
yes, that works